doc/docs/modules/ROOT/pages/quickstart_writer.adoc

```diff
@@ -215,4 +215,108 @@ UNwIND $events AS event
 CREATE (n:Person{fullName: event.name + event.surname})
 ----
 
-Where `event` represents each Dataset row.
\ No newline at end of file
+Where `event` represents each Dataset row.
+
+=== Write data by Relationship Type
+
+You can write a dataframe to Neo4j by specifying source, target and relation.
+There are two strategies you can use to write relationships: *NATIVE* and *KEYS*
+
+==== Native Strategy
+
+This strategy is useful when you have a schema that conforms with the <<quickstart_reader.adoc#bookmark-rel-read-schema,Relationship Read Schema>>
+
+```scala
+df.write
+    .format("org.neo4j.spark.DataSource")
+    .option("url", "bolt://localhost:7687")
+    .option("relationship", "SOLD")
+    .option("relationship.source.labels", ":Person")
+    .option("relationship.source.save.mode", "Overwrite")
+    .option("relationship.target.labels", ":Product")
+    .option("relationship.target.save.mode", "Overwrite")
+    .save()
+```
+
+You just need to specify the source node labels, the target node labels, and the relationship you want between them.
+
+[NOTE]
+The default save mode for source and target nodes is `match`. This means that the relationship will be created only if the nodes are already in your DB. Look at <<node-save-modes,here>> for more info about node save modes.
+
+Using `Overwrite` or `Match` node save mode, you can speficy which keys should be used to write the nodes.
+
+```scala
+df.write
+    .format("org.neo4j.spark.DataSource")
+    .option("url", "bolt://localhost:7687")
+    .option("relationship", "SOLD")
```

Comment:
labels doesn't require `:` if it's just one label.

Label list can be specified both with starting colon or without it:`:Person` and `Person` are considered the same thing.

Reply:
You've got great answers to these questions -- but don't tell me the answer.  Check it into the docs!  Can't figure out where it goes?  Put it in the FAQ and we'll sort it out into a different section later.  Otherwise this will be the first -- but not the last -- time you get asked the question and you'll get to type the answer 5x.   :)